.circleci/config.yml

```diff
@@ -49,10 +89,26 @@ jobs:
     steps:
       - build-artifacts
 
+  package-extension:
+    description: 'Packages extenson in a Windows environment'
+    executor: win/default
+    environment:
+      - SFDX_AUTOUPDATE_DISABLE: true
+      - SFDX_DOMAIN_RETRY: 300
+      - NODE_VERSION: '12.4.0'
```

Comment:
If you're going to enforce node version `12.4.0` you should update the engine vscode api in package.json from `^1.30.0` to `^1.40.0`. This is because version 1.30.0 of vscode runs on nodejs 8 (https://code.visualstudio.com/updates/v1_30#_continued-electron-30-exploration). They introduced node 12 in version 1.40.0.

If you want to keep the extension usable for vscode versions `1.30.0` and above then you should have your tests running in node 8, node 10 and node 12.

Reply:
Thanks for the feedback. I updated the vscode version in `package.json`